ports/jediacademy/JediAcademy/openjk_sp.aarch64.gptk

```diff
@@ -1,28 +1,28 @@
-back = esc
-start = enter
-a = space
-b = b
-x = x
-y = y
-l1 = mouse_right
-l2 = n
-l3 = g
-r1 = mouse_left
-r2 = m
-r3 = h
-guide = h
-up = up
-down = down
-left = left
-right = right
-left_analog_up = w
-left_analog_down = s
-left_analog_left = a
-left_analog_right = d
-right_analog_up = mouse_movement_up
-right_analog_down = mouse_movement_down
-right_analog_left = mouse_movement_left
-right_analog_right = mouse_movement_right
+back =  n                                       # Quick Load
+start = m                                       # Quick Save
+guide = esc                                     # Pause / Menu  
```

Comment:
On most devices there is no guide button. Guide is often synonymous with back. What button did you use on your rg35xxh to test this?

Reply:
Guide = Function
Back = Select